platform-dom/src/main/java/com/softicar/platform/dom/refresh/bus/DomRefreshBus.java

```diff
@@ -92,4 +90,35 @@ private void submitQueue() {
 		}
 		queue.clear();
 	}
+
+	private void convertDanglingNodeListeners() {
+
+		var bodyFinder = new DomParentNodeFinder<>(DomBody.class);
```

Comment:
- this is already the 2nd use of this check, IMO we should add something like `IDomNode.isAppendedToDocument()` and also use that in `DomDeferredInitializationController.java`
- please note that `isAppendedToDocument` is better than `isAppendedToBody`, because actually, the HTML standard defines a `document` element above the body element which we do not reflect in our tree because we don't need it

Reply:
I noticed that we already have `IDomNode.isAppended`, with an implementation in `AbstractDomNode` (which relies on `DomBody.isAppended` and `DomHead.isAppended` always returning `true`).
`DomRefreshBus` and `DomDeferredInitializationController` now use `IDomNode.isAppended`.